examples/demo/otel-collector-config.yaml

```
@@ -42,3 +42,11 @@ service:
       receivers: [otlp]
       processors: [batch]
       exporters: [debug, prometheus]
+  telemetry:
+    metrics:
+      readers:
+        - pull:
+            exporter:
+              prometheus:
+                host: '0.0.0.0'
+                port: 8888
```

```suggestion
                port: 8888

```
Nit: Remove whitespace change